packages/api/prisma/schema.prisma

```diff
@@ -64,8 +63,15 @@ model CourseProgress {
 }
 
 model EventsSubscription {
-    id                  String @id
-    currentPosition     Int
-    fromPosition        Int
-    eventTypes          String[]
+  id              String   @id
+  currentPosition Int
+  fromPosition    Int
+  eventTypes      String[]
+}
+
+model EventLock {
```

Comment:
Thank you 👍🏻 Event linking and db structure similar to RailsEventStore can be easily implemented after this PR. Check out [this proposal commit](https://github.com/CodersCrew/coderscamp/commit/23fe0c3a2b21c0d3ee68e0b5459bd7e2c1687cac).

Reply:
You're amazing! :) 
Maybe create library with EventStore instead of CodersCamp app :P ? 
I will look at that tommorow.

With Event linking we can eventually optimize subscriptions to read just events from a single stream, instead of one.